doc/chassis/architecture.md

```diff
@@ -0,0 +1,285 @@
+
+* A forwarding module must act as a fully functional router that can run routing protocols and other networking services just like single box SONiC devices. 
+* The system of forwarding modules should be managed by a single central Supervisor SONiC instance (SSI) that also manages the internal fabric that interconnects the forwarding modules.
+
+## 1.1.2 Intra-System Control Plane
+
+* Each FSI should be able to connect to other FSIs over the internal fabric in order to be able to run protocols like BGP within the system. 
+
+* This connection must be fate shared with the data path so that a loss of connectivity in the internal fabric is reflected as loss of internal control plane connectivity as well.
+
+## 1.1.3 Intra-System Management Plane
+
+Every FSI must have a management interface over which it can reach the supervisor and the rest of the network outside the system. This network must be completely separate from the internal control plane network described above.
+
+## 1.2 Configuration and Management Requirements
+
+* Each SONiC instance must be independently configurable and manageable through standard SONiC management interfaces.
+ 
+* The physical configuration of the entire system is fixed at startup. This includes
+  * The Hardware SKU that is used for each forwarding module
+  * The Physical port organization of the entire system
+
+* Live replacement of forwarding modules or pluggable modules like transceivers must be supported as long as the part being replaced is an identical SKU.
+
+# 2 Design
+
+## 2.1 Design Assumptions
+
+In order for the system to function correctly, some state that provides the global view of the system to all the FSIs is necessary. This state is stored in the SSI and all FSIs connect to the SSI over the internal management network to access this state.
+
+## 2.2 SAI Support
+
+Support for VOQ based forwarding in SONiC is dependent on the [SAI VOQ API](https://github.com/opencomputeproject/SAI/blob/master/doc/VoQ/SAI-Proposal-VoQ-Switch.md)
+
+## 2.3 State Sharing
+
+![](../../images/chassis/architecture.png)
+
+All state of global interest to the entire system is stored in the SSI in a new Redis instance with a database called “Global DB”. This instance is accessible over the internal management network. FSIs connect to this instance in addition to their own local Redis instance to access and act on this global state.
+
+##  2.3.1 Global DB Organization
+
+The Global DB is hosted in a new redis instance called `redis_global`. This new redis instance runs in a new container known as `docker-database-global`. This ensures both that the global state is isolated from the rest of the databases in the instance and can also be conditionally started only on the SSI.
+
+### 2.3.1.1 Starting redis_global in the SSI
+
+To detect if the `docker-database-global` container needs to be started, a new configuration file called `globaldb.conf` is introduced at `/usr/share/sonic/device/<platform>/globaldb.conf`. 
```

Comment:
during my codereview, Maxime suggested using global_db.conf instead globaldb.conf, just like /etc/sonic/config_db.conf, I feel it is reasonable.